### PR TITLE
Домашняя работа №3

### PR DESCRIPTION
**Pull request by Chepchikk**

### CODE REVIEW

File: homework_3.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+let item_7_type;
+item_7_type = item_7;
+console.log('item_7 == ' + typeof item_7, 'item_7_type == ' + typeof item_7_type); //Вывели в консоль типы данных
+
+
+//TASK-2 Решение квадратных уравнений
+
+//х2 - 6х + 9 = 0  D = b ** 2 - 4*a*c
+
+let a = 1;
```
Comment (anatoly-karpovich) [inline homework_3.js:42]: Let should be replaced with const for almost all variables in this code (except variables that are being changed)

File: homework_3.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+let x1 = (-b + Math.sqrt(D))/(2*a);
+let x2 = (-b - Math.sqrt(D))/(2*a);
+console.log('Ответ к уравнению 2: ' + x1 + ' и ' + x2);
+
+
+
+//TASK-3 n+nn+nnn
+
+let n;
+n = 3;
```
Comment (anatoly-karpovich) [inline homework_3.js]: Try let n = 3, looks better)

**Review by anatoly-karpovich — CHANGES_REQUESTED** [review]

File: homework_3.js (hunk 1 above)
Comment (Chepchikk) [reply homework_3.js:42]: let исправлены на const

File: homework_3.js (hunk 2 above)
Comment (Chepchikk) [reply homework_3.js]: Сделал лучше)

File: homework_3.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+x = (-b + Math.sqrt(D))/(2*a);
+console.log('Ответ к уравнению 1: ' + x);
+
+//x2 - 4x - 5 = 0
+
+a = 1;
+b = -4;
+c = -5;
+D = b**2 - 4*a*c; // Дискрименант = 0, соответсвенно 1 корень
+let x1 = (-b + Math.sqrt(D))/(2*a);
```
Comment (anatoly-karpovich) [inline homework_3.js]: let in tasks and 3 should be replaced with const too)

**Comment by anatoly-karpovich** [comment]
Please provide a brief description to your PR

**Review by anatoly-karpovich — CHANGES_REQUESTED** [review]

**Comment by Chepchikk** [comment]
Исправления

**Review by anatoly-karpovich — APPROVED** [review]

**Pull request merged**